subhub/customer.py

```diff
@@ -32,7 +32,9 @@ def create_customer(
         if possible_customer.email == email:
             # If the userid doesn't match, the system is damaged.
             if possible_customer.metadata.get("userid") != user_id:
-                raise ServerError("customer email exists but userid mismatch")
+                raise ServerError(  # type: ignore
```

Comment:
this is ugly; if black requires this, then maybe declare the str for the `customer email exists but userid mismatch` on a line a above